lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py

```diff
@@ -0,0 +1,561 @@
+    requested_service_objective_id:
+        description:
+            - "The configured service level objective ID of the database. This is the service level objective that is in the process of being applied to the
+               database. Once successfully updated, it will match the value of currentServiceObjectiveId property. If requestedServiceObjectiveId and
+               I(requested_service_objective_name) are both updated, the value of requestedServiceObjectiveId overrides the value of
+               I(requested_service_objective_name). To see possible values, query the capabilities API
+               (/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities) referred to by operationId:
+               'Capabilities_ListByLocation.'"
+    requested_service_objective_name:
+        description:
+            - "The name of the configured service level objective of the database. This is the service level objective that is in the process of being applied
+               to the database. Once successfully updated, it will match the value of serviceLevelObjective property. To see possible values, query the
+               capabilities API (/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities) referred to by operationId:
+                'Capabilities_ListByLocation.'."
+        choices:
+            - 'basic'
+            - 's0'
+            - 's1'
+            - 's2'
+            - 's3'
+            - 'p1'
+            - 'p2'
+            - 'p3'
+            - 'p4'
+            - 'p6'
+            - 'p11'
+            - 'p15'
+            - 'system'
+            - 'system2'
+            - 'elastic_pool'
+    elastic_pool_name:
+        description:
+            - "The name of the elastic pool the database is in. If elasticPoolName and I(requested_service_objective_name) are both updated, the value of
+               I(requested_service_objective_name) is ignored. Not supported for C(data_warehouse) edition."
+    read_scale:
+        description:
+            - "If the database is a geo-secondary, readScale indicates whether read-only connections are allowed to this database or not. Not supported for
+               C(data_warehouse) edition."
+        type: bool
+        default: False
+    sample_name:
+        description:
+            - "Indicates the name of the sample schema to apply when creating this database. If I(create_mode) is not C(default), this value is ignored. Not
+               supported for C(data_warehouse) edition."
+        choices:
+            - 'adventure_works_lt'
+    zone_redundant:
+        description:
+            - Is this database is zone redundant? It means the replicas of this database will be spread across multiple availability zones.
+        type: bool
+        default: False
+    force_update:
+      description:
+          - Needs to be set to True in order to SQL Database to be updated.
```

Comment:
It seems like this is only the case for certain objects, might be best to explain this is more detail, also what is the default option.